Documentation/ksmbd.adduser.8

```diff
@@ -0,0 +1,35 @@
+.TH KSMBD.ADDUSER "8" "October 2021" "Usage: smbuseradd" "Linux System Administration"
+.SH NAME
+ksmbd.adduser \- Adds, updates or removes (-a/-u/-d) a user from ksmbd pwd file.
+.SH SYNOPSIS
+.B ksmbd.adduser
+
+.SH DESCRIPTION
+.HP
```

Comment:
We don't need to update any description as per command ?

Reply:
I guess I will work on improving the manual pages until I'm happy with them, before I submit them where you asked them to be sent to. Thanks so far.

Reply:
Okay, there is no description in some option of command. I guess you want that I need to help you add a description of the command's options in manpage. Right ? or you will do that after checking ksmbd-tools code ?

Reply:
ahh true the descriptions are completely missing, they only list the parameters and what they do. so that needs be filled out. personally i'd like to add an EXAMPLES section, so no one gets the idea to create tldr pages. since i've had problems getting the ksmbd kernel module debian dkms packaged right (but it builds and loads fine), my plan was to wait for 5.15 kernel, so i'll get it right in the kernel, then test and improve the manual pages later, but if you got spare time, please go ahead, you can edit everything as well after all. thank you for being faster?